client/src/components/Layout/Header.js

```diff
@@ -0,0 +1,161 @@
+          <Typography variant="h6" className={classes.title}>
+            calendapp
+          </Typography>
+            <div>
+              {isMobile ? (
+                <>
+                  <IconButton 
+                    edge="start" 
+                    className={classes.menuButton} 
+                    color="inherit" 
+                    aria-label="dashboard-mobile"
+                    onClick={() => handleButtonClick("/event_types/user/me")}>
+                    <Typography>Home</Typography>
+                  </IconButton>
+                  <IconButton
+                    aria-label="account of current user"
+                    aria-controls="menu-appbar-mobile"
+                    aria-haspopup="true"
+                    onClick={handleMenu}
+                    color="inherit"
+                  >
+                    <AccountCircle />
+                  </IconButton>
+                <Menu
+                  id="menu-appbar-mobile"
+                  anchorEl={anchorEl}
+                  anchorOrigin={{
+                    vertical: 'top',
+                    horizontal: 'right',
+                  }}
+                  keepMounted
+                  transformOrigin={{
+                    vertical: 'top',
+                    horizontal: 'right',
+                  }}
+                  open={open}
+                  onClose={() => setAnchorEl(null)}
+                >
+                  <MenuItem onClick={() => handleMenuClick("/profile")}>Account Settings</MenuItem>
+                  <MenuItem onClick={() => handleMenuClick("/profile")}>Availablity</MenuItem>
+                  <MenuItem onClick={() => handleMenuClick("/profile")}>Integration</MenuItem>
+                  <MenuItem onClick={() => handleMenuClick("/")}>Logout</MenuItem>
+                </Menu>
+                </>
+              ) : (
+                <>
+                  <IconButton
+                      aria-label="dashboard of current user"
+                      aria-controls="menu-appbar"
+                      aria-haspopup="true"
+                      onClick={() => handleButtonClick("/event_types/user/me")}
+                      color="inherit"
+                    >
+                      <Typography> Home </Typography>
+                  </IconButton>
+                  <IconButton
+                      aria-label="availability of current user"
+                      aria-controls="menu-appbar"
+                      aria-haspopup="true"
+                      onClick={() => handleMenuClick("/profile")}
+                      color="inherit"
+                    >
+                      <Typography> Availablity </Typography>
+                  </IconButton>
+                  <IconButton
+                      aria-label="integrations of current user"
+                      aria-controls="menu-appbar"
+                      aria-haspopup="true"
+                      onClick={() => handleMenuClick("/profile")}
```

Comment:
i wonder if you can change this into a `href`

```suggestion
                      href="/profile"
```